GPU/Common/SplineCommon.cpp

```diff
@@ -361,16 +383,45 @@ static void _SplinePatchFullQuality(u8 *&dest, u16 *indices, int &count, const S
 	}
 }
 
+template <bool origNrm, bool origCol>
+static inline void SplinePatchFullQualityDispatch3(u8 *&dest, u16 *indices, int &count, const SplinePatchLocal &spatch, u32 origVertType, int quality, int maxVertices) {
+	bool origTc = (origVertType & GE_VTYPE_TC_MASK) != 0;
```

Comment:
Thanks for the explain, I understood that is doing what.

> Not sure it's actually worth it, unknownbrackets thought so though.

Yeah, it's the point that I want to know.
Of course, I think I should respect his code and his huge contribution.
However, I want to cleanup spline/bezier codes to more clear, and improve the speed of software tessellation(I found the bottleneck). If the codes can't so benefits than more simple codes, I'll change this and entrust to your decision anyway.